src/ProjectNeon/Assets/Data/Templates/Character.cs

```diff
@@ -1,8 +1,14 @@
-﻿using UnityEngine;
+﻿using System.Collections.Generic;
+using UnityEngine;
 
 public class Character : ScriptableObject
 {
     public Sprite Bust;
     public Stats Stats;
     public string ClassName;
+
+    [SerializeField]
+    private List<Card> cards;
```

Comment:
Characters are supposed to be Design-Time unchangeable reference data.

Decks are mutable and will change during the game. `PartyDecks` is the state structure you are looking for.